ChineseFilter/sections/specific.txt

```diff
@@ -22,6 +22,8 @@ $script,xmlhttprequest,third-party,domain=00ksw.com|01zww.com|0biquge.com|114zw.
 !
 !
 v2ex.com##div[class^="wwads-"]
+healthylives.tw###contentBox > div[style="width:336px;height: 280px;"]
```

Comment:
Style is a bit different on my end. `^` is required.(`div[style^="width:336px;height: 280px;"]`)